### DISCUSSION TITLE
Can Somebody help me fix this error? Error: unsupported/unknown protocol version: 762, update minecraft-data

### DISCUSSION THREAD

**Discussion by turbomaster95** [PrismarineJS/mineflayer#3007] [Q&A]
I am making a bot for using on my aternos server.
But it is saying

`Error: unsupported/unknown protocol version: 762, update minecraft-data`
The screenshot :
![Screenshot_2023-04-03-21-13-57-09_84d3000e3f4017145260f7618db1d683](https://user-images.githubusercontent.com/86113389/229560653-aaf3890c-d4e5-49a8-b2e2-5bd869efc750.jpg)

I am using Termux on Android 11 ( i think?..)

**Comment by turbomaster95** [comment]
Please help me

**Reply by amoraschi** [reply]
I recommend reading the error, might give a hint

![Screenshot_2023-04-03-21-12-41-116_com.github.android-edit.jpg](https://user-images.githubusercontent.com/68395930/229604657-43f54197-30e0-4b5a-8308-c227ba5e3890.jpg)
